File: capi/src/main/java/org/roylance/yadel/services/IReportService.kt
package org.roylance.yadel.services

interface IReportService {
	fun delete_dag(request: org.roylance.yadel.YadelReport.UIYadelRequest): org.roylance.yadel.YadelReport.UIYadelResponse
	fun current(request: org.roylance.yadel.YadelReport.UIYadelRequest): org.roylance.yadel.YadelReport.UIYadelResponse
}

File: javascript/IReportService.ts
export interface IReportService {
	delete_dag(request: org.roylance.yadel.UIYadelRequest, onSuccess:(response: org.roylance.yadel.UIYadelResponse)=>void, onError:(response:any)=>void)
	current(request: org.roylance.yadel.UIYadelRequest, onSuccess:(response: org.roylance.yadel.UIYadelResponse)=>void, onError:(response:any)=>void)
}